Orange/statistics/util.py

```diff
@@ -193,24 +193,23 @@ def stats(X, weights=None, compute_variance=False):
     is_sparse = sp.issparse(X)
     weighted = weights is not None and X.dtype != object
 
-    if weighted:
-        weights = np.c_[weights] / sum(weights)
```

Comment:
I'd still compute `weights` instead of doing it in two places later. Just a suggestion, feel free to ignore.